github/tests/ReplayData/Repository.testGetTopics.txt

```diff
@@ -0,0 +1,10 @@
+https
```

Comment:
Maybe Github has updated the API, but for this repo the GET response is simply:
```json
{
  "names": [
    "pygithub",
    "python",
    "github",
    "github-api"
  ]
}
```

Reply:
That is the expected output based on the API:  https://developer.github.com/v3/repos/#response-5